src/main/java/card/information/CreditCardInformation.java

```diff
@@ -0,0 +1,21 @@
+package main.java.card.information;
+
+
+public class CreditCardInformation {
+
+    public CreditCardInformation(String[] inputArgs){
```

Comment:
In the constructor, it is better to pass two variables explicitly, so that the responsibility for their selection is not on the class's side
```suggestion
    public CreditCardInformation(String cardIssuerName, String cardNumber){
        this.cardIssuerName = cardIssuerName;
        this.cardNumber = cardNumber;
   }
   
```